ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/PackageConfigurationFactory.kt

```diff
@@ -222,11 +223,20 @@ internal object PackageConfigurationFactory {
                 it.product.pricePerMonth(),
                 mostExpensivePricePerMonth,
             )
+
+            val shouldRound = if (Purchases.isConfigured) {
```

Comment:
Hmm I might be missing something but is this check needed? I would assume that if we have `paywallData` at all, the SDK should be configured?

Reply:
Ya, this value should be passed in from elsewhere. The issue was with a number of tests that hit his method without configuring Purchases.